eledpar.dtx

```diff
@@ -605,11 +605,11 @@
 %
 % \subsection{\cs{AtEveryPstart} and \cs{AtEveryPstartCall}}
 % In general, remember that the moment where a \cs{pstart} is called is different 
-% from the moment when  the \cs{pstart}…\cs{pend} content is printed, which is when \cs{Pages} or \cs{Columns} is processed.
+% from the moment when  the \cs{pstart}\ldots\cs{pend} content is printed, which is when \cs{Pages} or \cs{Columns} is processed.
 % 
 % Consequently:
 % \begin{itemize}
-%  \item The argument of \cs{AtEveryPstart} (see \reff{eledmac-AtEveryPstart}) is called before every chunk printing, except if you use an optional argument for the \cs{pstart}.
+%  \item The argument of \cs{AtEveryPstart} (see \reff{eledmac-AtEveryPstart}) is called before every chunk is printed, except if you used an optional argument for the \cs{pstart}.
```

Comment:
> > >  % 
> > >  % Consequently:
> > >  % \begin{itemize}
> > > -%  \item The argument of \cs{AtEveryPstart} (see \reff{eledmac-AtEveryPstart}) is called before every chunk printing, except if you use an optional argument for the \cs{pstart}.
> > > +%  \item The argument of \cs{AtEveryPstart} (see \reff{eledmac-AtEveryPstart}) is called before every chunk is printed, except if you used an optional argument for the \cs{pstart}.
> > 
> > before each chunk is printed est plus pr�cis. je change�
> > 
> ok
Oui, mais comme la commande s'appelle AtEveryPstart, c'est embêtant...
On est un peu dans les subtilités entre "tout" (every) et "chaque"
(each)... Quand on dit: "every room has a view of the sea", on peut
certes traduire "chaque chambre a une vue sur la mer", mais on
comprend toujours que toutes les chambres ont en fait vue sur la mer.

Each, c'est différent. Un seul exemple pour le montrer, à partir du
premier: "every room has a view of the sea, and each more beautiful
than the other": on voit bien que each les prend une par une.

C'était le sens de ma remarque initiale...